exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -345,11 +347,55 @@ void createBlockWithTransactions() {
     }
   }
 
+  @Test
+  void nodeSubmittedTransactionsArePlacedInPool() {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+
+      TransactionMessage message = constructTestTransactionMessage("Test message", testKit);
+      RawTransaction rawTransaction = TestKit.toRawTransaction(message);
+
+      try {
+        service.getNode().submitTransaction(rawTransaction);
+      } catch (InternalServerError e) {
+        fail(e);
```

Comment:
Just add `throws Exception` — JUnit will mark this test as an error in case it throws an exception.